app/manifest/v2/_base.json

```diff
@@ -64,14 +64,11 @@
     "storage",
     "unlimitedStorage",
     "clipboardWrite",
-    "http://localhost:8545/",
-    "https://*.infura.io/",
-    "https://*.codefi.network/",
-    "https://*.cx.metamask.io/",
-    "https://chainid.network/chains.json",
-    "https://lattice.gridplus.io/*",
+    "http://*/*",
+    "https://*/*",
     "activeTab",
     "webRequest",
+    "webRequestBlocking",
```

Comment:
The question: does this cause permissions warnings to FF users (and any other MV2 browsers we build for)?

Reply:
Instructions here: https://extensionworkshop.com/documentation/develop/test-permission-requests/

I can test this

Reply:
I was able to test this. I created three branches:
* `manifest-update-test-1`
* `manifest-update-test-2`
* `manifest-update-test3`.

I pushed the manifest changes I made to those branches. I used a local update server, which I ran using the command `./development/static-server.js -p 9999 ./firefox-updates`. The `firefox-updates` directory was a local directory I created with the XPI files, and an `updates.json` file that had the relevant updates.

I used Firefox Developer Edition, but I had to follow some Nightly-specific instructions in that doc (I guess it's inaccurate in suggesting they're only needed on nightly). It was the instructions about setting things in `about:config` to enable a local update server.

Here was the final `updates.json` file I used:

```
{
  "addons": {
    "webextension@metamask.io": {
      "updates": [
        {
          "version": "11.16.7",
          "update_link": "http://localhost:9999/metamask-firefox-11.16.7.xpi"
        },
        {
          "version": "11.16.8",
          "update_link": "http://localhost:9999/metamask-firefox-11.16.8.xpi"
        },
        {
          "version": "11.16.9",
          "update_link": "http://localhost:9999/metamask-firefox-11.16.9.xpi"
        }
      ]
    }
  }
}
```

I created a build from each branch using `ENABLE_MV3 yarn list`. The first branch was based on `develop`, the second was based on this branch, and the third was also based on this branch but had the additional `tabs` permission (which should result in a warning on update). Each branch had the `update_url` manifest change and a version bump as well.

I first removed `11.16.9` from `updates.json` to test this branch, and no warning appeared. Then I added `11.16.9` back, and I did get a warning about the tabs permission being added.

We should be safe to add this permission, there was no additional warning.